src/components/ShopByCategory/ShopByCategory.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Grid from "@mui/material/Grid";
+import './ShopByCategory.scss';
+import { CategoryCard } from '../CategoryCard';
+
+import categoryPhonesImg from "../../img/categories/category-phones.jpg";
+import categoryTabletsImg from "../../img/categories/category-tablets.jpg";
+import categoryAccessoriesImg from "../../img/categories/category-accessories.jpg";
+
+export const ShopByCategory: React.FC = () => {
+  const phonesAmount = 95;
```

Comment:
i dont know better this or not, not use hardcoded value, but get array length from server and display it(